contrib/packs/actions/deploy.yaml

```diff
@@ -19,11 +19,11 @@
     branch:
       type: "string"
       description: "The Branch to deploy."
-      required: true
+      default: "master"
     packs:
       type: "array"
       description: "The Pack(s) to deploy."
-      required: true
+      default: []
```

Comment:
We talked about his on Slack, this is so that it's possible to install a pack which has no sub tree with out supplying the same string for repo_name and packs.